validation-test/stdlib/StringSlicesConcurrentAppend.swift

```diff
@@ -52,12 +59,15 @@ func sliceConcurrentAppendThread(tid: ThreadID) {
     barrier()
 
     // Get a private string.
+    // we *should* be able to change this as we wish without modifying the underlying string.
     var privateString = sharedString
 
     barrier()
 
     // Append to the private string.
-    if tid == .Leader {
+    // This should invoke a deep copy somewhere along the line,
```

Comment:
Swift does not use or define "deep copy", so I don't know what it means in this context.